packages/block-editor/src/components/block-list/index.js

```diff
@@ -47,7 +62,24 @@ function Root( { className, ...settings } ) {
 		},
 		[]
 	);
+	const registry = useRegistry();
 	const { setBlockVisibility } = useDispatch( blockEditorStore );
+
+	const delatedBlockVisibilityUpdates = useDebounce(
```

Comment:
```suggestion
	const delayedBlockVisibilityUpdates = useDebounce(
```

😁 I actually searched what [delated](https://www.dictionary.com/browse/delated) means

Reply:
😂